lint/unusedParams_checker.go

```diff
@@ -0,0 +1,55 @@
+package lint
+
+import (
+	"fmt"
+	"go/ast"
+)
+
+func init() {
+	addChecker(unusedParamsChecker{}, &ruleInfo{})
+}
+
+type unusedParamsChecker struct {
```

Comment:
Can we call it `unusedParam` (without "s")?

Reply:
sure